xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/editors/dataeditors.js

```diff
@@ -753,7 +753,9 @@ editors.XDataEditors = Class.create({
       var movebutton = new Element('span', {
         'class': 'tool move',
         title: 'Drag and drop to change the order'
-      }).update('move');
+      }).update('<span class="presentation">' +
+        $jsontool.serialize($services.icon.renderHTML('arrows')) + 
+        '</span><span class="sr-only">move</span>');
```

Comment:
Do we have an issue that this should be translated? If not, could you create one?

Reply:
I figured the sr-only span is not necessary. I removed it and added a translation to the 'button' title where this icon appears. :+1: